cli/tests/it/cast.rs

```diff
@@ -240,6 +240,43 @@ casttest!(cast_run_succeeds, |_: TestProject, mut cmd: TestCommand| {
     assert!(!output.contains("Revert"));
 });
 
+// tests that the `cast storage` command works correctly
+casttest!(cast_storage_succeeds, |_: TestProject, mut cmd: TestCommand| {
```

Comment:
For some reason the etherscan key is not read here?

Reply:
from what I'm seeing in other tests, they only run when the api key/private key env var is set (https://github.com/foundry-rs/foundry/blob/master/cli/tests/it/verify.rs)
and they're set only in the live tests

Reply:
@mattsse let's bring the etherscan api key in scope here? Any reason not to?

Reply:
don't think so, but unsure if the required key secrets are set. will check and enable

Reply:
@mattsse when you get a chance, let's set the secrets here and merge

Reply:
I don't think I've access to set secrets.